plugin/neomux.vim

```diff
@@ -12,72 +12,89 @@ let s:os = systemlist("uname -s")[0]
 let s:arch = systemlist("uname -m")[0]
 
 function! s:NeomuxMain()
-    " Set default key bindings if no user defined maps present
-    if !exists('g:neomux_start_term_map') | let g:neomux_start_term_map = '<Leader>sh' | endif
-    if !exists('g:neomux_start_term_split_map') | let g:neomux_start_term_split_map = '<C-w>t' | endif
-    if !exists('g:neomux_start_term_vsplit_map') | let g:neomux_start_term_vsplit_map = '<C-w>T' | endif
-    if !exists('g:neomux_winjump_map_prefix') | let g:neomux_winjump_map_prefix = "<C-w>" | endif
-    if !exists('g:neomux_winswap_map_prefix') | let g:neomux_winswap_map_prefix =  "<Leader>s" | endif
-    if !exists('g:neomux_yank_buffer_map') | let g:neomux_yank_buffer_map = "<Leader>by" | endif
-    if !exists('g:neomux_paste_buffer_map') | let g:neomux_paste_buffer_map = '<Leader>bp' | endif
-    " Pnemonic: size-fix. Resize terminal window back to proper size
-    if !exists('g:neomux_term_sizefix_map') | let g:neomux_term_sizefix_map = '<Leader>sf'  | endif
-    if !exists('g:neomux_win_num_status') | let g:neomux_win_num_status = '∥ W:[%{WindowNumber()}] ∥' | endif
-    if !exists('g:neomux_exit_term_mode_map') | let g:neomux_exit_term_mode_map = '<C-s>' | endif
-    if !exists('g:neomux_default_shell') | let g:neomux_default_shell = "" | endif
-    if !exists('g:neomux_dont_fix_term_ctrlw_map') | let g:neomux_dont_fix_term_ctrlw_map = 0 | endif
-    if !exists('g:neomux_no_term_autoinsert') | let g:neomux_no_term_autoinsert = 0 | endif
-
     command! Neomux call NeomuxTerm()
 
     call NeomuxAddWinNumLabels()
 
     " Make getting out of terminal windows work the same way it does for every
     " other window. If you need to send a <C-w> keystroke to the term window,
     " use `:call NeomuxSendCtrlW()`
-    if g:neomux_dont_fix_term_ctrlw_map == 0
+    if !get(g:, 'neomux_dont_fix_term_ctrlw_map', 0)
         tnoremap <C-w> <C-\><C-n><C-w>
     endif
 
     " Automatically start insert mode when entering a terminal buffer
     " from: https://github.com/neovim/neovim/issues/8816#issuecomment-410502364
-    if g:neomux_no_term_autoinsert == 0
+    " TODO: document
```

Comment:
BTW, I felt like the help file was generated out of the README, or perhaps it was the other way around. Anyhow, this little setting wasn't documented which is what made me look in the source code in the first place :). Anyway, I'd like to contribute documentation to this and the other setting I added (`neomux_no_term_leave_after_exit`) but I would like it to be reproducible in case you do use such a conversion engine.

Speaking of docs, I know there's https://github.com/google/vimdoc which I never tried my self but it looks promising so it might be interesting to try it out here, as I'd imagine with this changes, it'll be even easier to get lost when documenting the options :).

Reply:
It sure was, I used this script to do the conversion: https://github.com/xolox/vim-tools.

vimdoc looks interesting, and I like the idea of having one authoritative place for the docs. If you'd be interested in converting this to use vimdoc I'd welcome the PR. If not, then updating the readme would be plenty, I can take care of re-generating the help file after.